### ISSUE TITLE
[Bug] App Crash: NullPointerException

### ISSUE THREAD

**Issue by Seykqo** [hushenghao/AndroidEasterEggs#266]
```
Device: SM-S908B (samsung - b0s), SDK: 34 (14), App: 2.5.2 (252)

java.lang.NullPointerException: Attempt to read from field 'F0.O F0.G.L' on a null object reference in method 'void F0.G.J(F0.G)'
	at F0.G.J(SourceFile:1)
	at F0.G.P(SourceFile:20)
	at F0.w0.g(SourceFile:5)
	at V.x.a(SourceFile:11)
	at V.H.h0(SourceFile:26)
	at V.a.f0(SourceFile:3)
	at U.u.g(SourceFile:51)
	at U.u.e(SourceFile:6)
	at U.s0.g(SourceFile:188)
	at G0.g0.doFrame(SourceFile:7)
	at G0.e0.doFrame(SourceFile:48)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1687)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1698)
	at android.view.Choreographer.doCallbacks(Choreographer.java:1153)
	at android.view.Choreographer.doFrame(Choreographer.java:1069)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1646)
	at android.os.Handler.handleCallback(Handler.java:958)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:230)
	at android.os.Looper.loop(Looper.java:319)
	at android.app.ActivityThread.main(ActivityThread.java:8919)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:578)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
	Suppressed: Q5.f: [G0.h0@1e5b53d, G0.K0@634f132, n0{Cancelling}@4b41e83, f0@e4f1800]

```

**Comment by hushenghao** [comment]
App: 2.5.2 (252)? 
I just released version 2.5.1 (41) yesterday

**Comment by hushenghao** [comment]
```
java.lang.NullPointerException: Attempt to read from field 'F0.O F0.G.L' on a null object reference in method 'void F0.G.J(F0.G)'
	at androidx.compose.ui.node.LayoutNode$WhenMappings.J(LayoutNode.kt:1)
	at androidx.compose.ui.node.LayoutNode$WhenMappings.P(LayoutNode.kt:20)
	at androidx.compose.ui.node.TreeSet.g(JvmTreeSet.jvm.kt:5)
	at androidx.compose.runtime.changelist.Operation$RemoveNode.execute(Operation.kt:468)
	at androidx.compose.runtime.changelist.Operations.h0(Operations.kt:26)
	at androidx.compose.runtime.changelist.ChangeList.f0(ChangeList.kt:3)
	at androidx.compose.runtime.CompositionLocalMap$Companion.g(CompositionLocalMap.kt:51)
	at androidx.compose.runtime.CompositionLocalMap$Companion.e(CompositionLocalMap.kt:6)
	at androidx.compose.runtime.Recomposer$runRecomposeAndApplyChanges$2.g(Recomposer.kt:188)
	at androidx.compose.ui.platform.AndroidUiFrameClock$withFrameNanos$2$callback$1.doFrame(AndroidUiFrameClock.android.kt:41)
	at androidx.compose.ui.platform.AndroidUiDispatcher.performFrameDispatch(AndroidUiDispatcher.android.kt:109)
	at androidx.compose.ui.platform.AndroidUiDispatcher.access$performFrameDispatch(AndroidUiDispatcher.android.kt:41)
	at androidx.compose.ui.platform.AndroidUiDispatcher$dispatchCallback$1.doFrame(AndroidUiDispatcher.android.kt:69)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1687)
	at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1698)
	at android.view.Choreographer.doCallbacks(Choreographer.java:1153)
	at android.view.Choreographer.doFrame(Choreographer.java:1069)
	at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:1646)
	at android.os.Handler.handleCallback(Handler.java:958)
	at android.os.Handler.dispatchMessage(Handler.java:99)
	at android.os.Looper.loopOnce(Looper.java:230)
	at android.os.Looper.loop(Looper.java:319)
	at android.app.ActivityThread.main(ActivityThread.java:8919)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:578)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1103)
	Suppressed: Q5.f: [G0.h0@1e5b53d, G0.K0@634f132, n0{Cancelling}@4b41e83, f0@e4f1800]
```
Maybe it's because you modified something, because I can't restore the stack through mapping.txt
